src/lib/serviceRoutinesV2/postSubscriptions.cpp

```diff
@@ -62,7 +62,48 @@ extern std::string postSubscriptions
     return answer;
   }
 
-  TIMED_MONGO(ciP->httpStatusCode = mongoSubscribeContext(&parseDataP->scr.res, &scr, ciP->tenant, ciP->uriParam, ciP->httpHeaders.xauthToken, ciP->servicePathV));
+
+  //
+  // String filter in URI param 'q' ?
+  // What if it was also in NotifyConditionVector::NotifyCondition::Expression ... ?
+  //
+  if (ciP->uriParam[URI_PARAM_Q] != "")
```

Comment:
q parameter works only for `GET /v2/entities`. In the case of subscriptions the q parameter is part of the payload, already parsed by parsing stage (I guess) and put inside a SubscriptionExpression. Thus, I think that no special processing has to be done in serviceRoutinesV2/postSubscriptions.cpp.